lua/plenary/path.lua

```diff
@@ -158,7 +158,24 @@ local check_self = function(self)
   return self
 end
 
-Path.__index = Path
+Path.__index = function(t, k)
+  local raw = rawget(Path, k)
+  if raw then
+    return raw
+  end
+
+  if k == "_cwd" then
+    local cwd = uv.fs_realpath "."
+    t._cwd = cwd
+    return cwd
+  end
+
+  if k == "_absolute" then
+    local cwd = uv.fs_realpath(t.filename)
+    t._cwd = cwd
+    return cwd
```

Comment:
shouldn't this be `t._absolute`?

Otherwise lgtm Thanks :)

Reply:
Oh je, fixed :laughing: